PLATEXP-1812: Add RAM to a server or hypervisor (#1430)
rackerlabs/support-how-to: content/how-to/dedicated-hosting/reset-user-password-on-active-directory-domain/index.md

1. Log in to the [MyRackspace Portal](https://login.rackspace.com/login) with
   your username and password.

   <img width="409" alt="enterlogincredentials" src="/support/how-to/reset-user-password-on-active-directory-domain/resetpassword1.png">

> 2. In the top navigation bar, choose **Support** > **Support Center**.

Reviewer: ```suggestion
2. In the top navigation bar, click **Support** > **Support Center**.
```

Author: Update in PLATEXP 1735

   <img width="941" alt="selectsupportcenter" src="/support/how-to/reset-user-password-on-active-directory-domain/resetpassword2.png">

> 3. Under **Common Request** menu, click **See All Templates**. The **See All** Templates displays list of all templates.

Reviewer: ```suggestion
3. Under **Common Request** menu, click **See All Templates** and it will  display the list of all templates.
```

Author: Update in PLATEXP 1735

   <img width="958" alt="listoftemplates" src="/support/how-to/reset-user-password-on-active-directory-domain/resetpassword3.png">

> 4. From the **Common Request** menu, select **Reset User Password on Active Directory Domain** as shown in following image.

Reviewer: ```suggestion
4. From the **Common Request** window, select **Reset User Password on Active Directory Domain**.
```

Author: Update in PLATEXP 1735

   automatically for this user and provides it in the ticket. You can change
   this password later.

7. Click **Submit** button.

   After the ticket generates, you see your ticket list in the MyRackspace Portal.

>    It typically takes a few minutes for the ticket creation process to complete.

Reviewer: ```suggestion
   It takes a few minutes for the ticket creation process to complete.
```

Author: Update in PLATEXP 1735
